src/components/resource/AdminMenu.jsx

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+import React from 'react/addons';
+import {DropdownButton, MenuItem} from 'react-bootstrap';
+import classNames from 'classnames';
+
+import 'font-awesome/css/font-awesome.css';
+import './AdminMenu.scss';
+
+
+const AdminMenu = React.createClass({
+
+  displayName: 'AdminMenu',
+
+  propTypes: {
+    className: React.PropTypes.string,
+    resourceText: React.PropTypes.shape({
+      text: React.PropTypes.string,
+      description: React.PropTypes.string,
+      editUrl: React.PropTypes.string,
+    }).isRequired,
+    isAdmin: React.PropTypes.bool,
+  },
+
+  render() {
```

Comment:
Use [destructuring](http://facebook.github.io/react/docs/transferring-props.html#transferring-with-...-in-jsx) to extract useful props, and pass the rest to the element. See [HorizontalGauge::render](https://github.com/botify-labs/react-components/blob/devel/src/components/gauges/HorizontalGauge.jsx#L59).